Library/Homebrew/dev-cmd/man.rb

```diff
@@ -18,6 +18,10 @@ def man
     else
       regenerate_man_pages
     end
+
+    git_diff_args = ["-C", HOMEBREW_REPOSITORY, "diff", "--stat"]
+    git_diff_args << "--exit-code" if ARGV.flag? "--exit-code"
+    exit system "git", *git_diff_args, "docs/brew.1.html", "manpages"
```

Comment:
I think we can avoid showing any `git diff` output and just saying "no changes made" or something and setting the exit code accordingly to `--exit-code`.